src/runtime_src/core/edge/skd/main.cpp

```diff
@@ -34,65 +38,85 @@
  * and exit.
  */
 
+using severity_level = xrt_core::message::severity_level;
+
 int main(int argc, char *argv[])
 {
-  pid_t pid, sid;
-  xclDeviceHandle handle;
-  xclSKCmd cmd;
-
-  pid = fork();
-  if (pid < 0) {
+  pid_t pid = fork();
+  if (pid < 0)
     exit(EXIT_FAILURE);
-  }
+
   if (pid > 0)
     exit(EXIT_SUCCESS);
 
+  // Starting the child process
   umask(0);
 
-  /* Open syslog and send the first message */
-  openlog("skd", LOG_PID | LOG_CONS, LOG_LOCAL0);
-  syslog(LOG_INFO, "Daemon Start...\n");
+  // Send the first message to XRT log
+  const std::string msg = "Daemon Start...";
+  xrt_core::message::send(severity_level::info, "SKD", msg);
```

Comment:
You may not need a std::string here. I suspect const char * should work fine here.

Reply:
Yep, that is what happens with "const auto" ;)